react-ui/src/components/SignUpForm.js

```diff
@@ -37,52 +45,120 @@ handleFormChange = event => {
   console.log(stateName, val);
 };
 
-handleSubmit(e) {
-    e.preventDefault();
+signup = event => {
```

Comment:
passed in required parameters for integration and updated variable names to lowerCamel case for consistency.

Reply:
Thank you for the attention to detail, Seher!